updater/requirements.txt

```diff
@@ -1,3 +1,3 @@
-python-rtmidi==1.5.0
-spsdk==1.8.0
```

Comment:
Is this accidental or intentional? I think, if possible, it's better if we keep some version locking.

Reply:
it's intentional to get it to work on Mac, as we need later versions of both. We can pin them to the current versions, though.

It's not related to this pull request but was accidentaly included.